features/support/helpers.rb

```diff
@@ -0,0 +1,8 @@
+def prepare_env_hash_for(cmd)
+  if cmd.start_with? 'zat create'
+    path = File.expand_path('../../support/webmock', __FILE__)
+    { 'RUBYOPT' => "-r #{path}" }
```

Comment:
yes, we lost this information when we chose to extract it.

`prepare_env_hash_for_stubbing_requests_in_zat_create` ?